modules/renderer/map.js

```diff
@@ -217,8 +217,6 @@ export function Map(context) {
     function redraw(difference, extent) {
         if (!surface || !redrawEnabled) return;
 
-        clearTimeout(timeoutId);
```

Comment:
I thought about it a little more, and it probably needs to be:
`if (!difference && !extent) { queueRedraw.cancel(); }`
Because a partial redraw shouldn't cancel a full redraw.

Maybe `queueRedraw` could be named `debouncedFullRedraw` to make it clear. (sorry, I actually hate nitpicking on variable names in PRs).